static/app/utils/replays/hooks/useReplayData.tsx

```diff
@@ -190,40 +189,40 @@ function useReplayData({replaySlug, orgSlug}: Options): Result {
   );
 
   const fetchErrors = useCallback(
-    async errorIds => {
-      const conditions = new MutableSearch('');
-      errorIds.forEach((errorId, i) => {
-        if (i > 0) {
-          conditions.addOp('OR');
-        }
-        conditions.addFilterValues('id', [errorId]);
-      });
+    async (replayRecord: ReplayRecord) => {
+      if (!replayRecord.errorIds.length) {
+        return [];
+      }
 
       const response = await api.requestPromise(`/organizations/${orgSlug}/events/`, {
         query: {
           field: ['id', 'error.value', 'timestamp', 'error.type', 'issue.id'],
-          projects: [projectSlug],
-          query: conditions.formatString(),
+          projects: [-1],
+          start: replayRecord.startedAt.toISOString(),
+          end: replayRecord.finishedAt.toISOString(),
+          query: `id:[${String(replayRecord.errorIds)}]`,
+          referrer: 'api.replay.details-page',
```

Comment:
Where is this used?

Reply:
Its tagged on the backend & snuba creates transactions based on this referrer value